test/base/EtherCollateralLoansRepayTest.js

```diff
@@ -45,7 +44,6 @@ contract('EtherCollateralLoansRepayTest', function (accounts) {
             lendingPoolInstance.address,
             loanTermsConsInstance.address,
             settingsInstance.address,
-            collateralTokenInstance.address,
             atmSettingsInstance.address,
```

Comment:
I  think  the last param in the Loans.initialize is the collateral token.

Reply:
Changed